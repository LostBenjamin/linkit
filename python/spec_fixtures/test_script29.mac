
%include test_script29_include
%allow-mutability

$value 2

$OVERRIDE1 30
$OVERRIDE2 `30`
$OVERRIDE3 `<value>*30`

[m1]
  <OVERRIDE1>

[m2]
  <OVERRIDE2>

[m3]
  <OVERRIDE3>

