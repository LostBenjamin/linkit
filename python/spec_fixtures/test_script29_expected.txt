4:set:30
5:set:30
6:set:60

